light/rpc/client.go

```diff
@@ -581,6 +581,7 @@ func (c *Client) Tx(ctx context.Context, hash []byte, prove bool) (*ctypes.Resul
 // ProveShares calls rpcclient#ProveShares method and returns an NMT proof for a set
 // of shares, defined by `startShare` and `endShare`, to the corresponding rows.
 // Then, a binary merkle inclusion proof from the latter rows to the data root.
+// Deprecated. Use ProveSharesV2 instead.
```

Comment:
```suggestion
// Deprecated: use ProveSharesV2 instead.
```

Ref: 
- https://go.dev/wiki/Deprecated
- https://stackoverflow.com/questions/7849663/how-do-you-mark-code-as-deprecated-in-go